src/script/page/RightSidebar/GroupParticipantService/GroupParticipantService.tsx

```diff
@@ -0,0 +1,146 @@
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import {FC, useEffect} from 'react';
+
+import Icon from 'Components/Icon';
+import ServiceDetails from 'Components/panel/ServiceDetails';
+
+import {t} from 'Util/LocalizerUtil';
+import {matchQualifiedIds} from 'Util/QualifiedId';
+
+import PanelHeader from '../PanelHeader';
+
+import {UserState} from '../../../user/UserState';
+import {useKoSubscribableChildren} from 'Util/ComponentUtil';
+import {generatePermissionHelpers} from '../../../user/UserPermission';
+import {Conversation} from '../../../entity/Conversation';
+import {ActionsViewModel} from '../../../view_model/ActionsViewModel';
+import {IntegrationRepository} from '../../../integration/IntegrationRepository';
+import {User} from '../../../entity/User';
+import {ServiceEntity} from '../../../integration/ServiceEntity';
+import {handleKeyDown} from 'Util/KeyboardUtil';
+
+interface GroupParticipantServiceProps {
+  activeConversation: Conversation;
+  actionsViewModel: ActionsViewModel;
+  integrationRepository: IntegrationRepository;
+  onBack: () => void;
+  onClose: () => void;
+  serviceEntity: ServiceEntity;
+  userEntity: User;
+  userState: UserState;
+  isAddMode?: boolean;
+}
+
+const GroupParticipantService: FC<GroupParticipantServiceProps> = ({
+  activeConversation,
+  actionsViewModel,
+  integrationRepository,
+  onBack,
+  onClose,
+  serviceEntity,
+  userEntity,
+  userState,
+  isAddMode = false,
+}) => {
+  // TODO: Should be added (?)
+  // getLogger('GroupParticipantServiceViewModel');
+  const {
+    inTeam,
+    isActiveParticipant,
+    participating_user_ids: participatingUserIds,
+  } = useKoSubscribableChildren(activeConversation, ['inTeam', 'isActiveParticipant', 'participating_user_ids']);
+  const {self: selfUser} = useKoSubscribableChildren(userState, ['self']);
+  const {teamRole} = useKoSubscribableChildren(selfUser, ['teamRole']);
+
+  const {canChatWithServices, canUpdateGroupParticipants} = generatePermissionHelpers(teamRole);
+
+  const selectedInConversation = participatingUserIds.some(user => matchQualifiedIds(userEntity, user));
+
+  const showActions = isActiveParticipant && selectedInConversation && inTeam;
+
+  const onOpen = () => {
+    actionsViewModel.open1to1ConversationWithService(serviceEntity);
+  };
+
+  const onRemove = () => {
+    actionsViewModel.removeFromConversation(activeConversation, userEntity);
+    // this.onGoBack();
+  };
+
+  const onAdd = () => {
+    integrationRepository.addService(activeConversation, serviceEntity);
+    // this.onGoToRoot();
+  };
+
+  useEffect(() => {
+    integrationRepository.addProviderNameToParticipant(serviceEntity);
+  }, [serviceEntity]);
+
+  return (
+    <div id="group-participant-service" className="panel__page group-participant panel__page--visible">
+      <PanelHeader onGoBack={onBack} goBackUie="go-back-group-participant" onClose={onClose} />
+
```

Comment:
Please ask Astrid what should be the values for this properties like I have added in NotificationsPanel
goBackTitle, title, closeBtnTitle

@przemvs